SConstruct

```diff
@@ -3,20 +3,12 @@ import os
 
 from lsst.sconsUtils import scripts
 from lsst.sconsUtils.state import env
+from lsst.sconsUtils import targets
```

Comment:
Merged into the above
```
from lsst.sconsUtils import scripts, targets
```
I wouldn't object if you ran `isort` and `black`/`ruff` on this entire script either; there are quite a lot of formatting errors.